scripts/lxc-pstart

```diff
@@ -221,84 +234,77 @@ def get_psuedo_init_blob():
         return fp.read(), fpath
 
 
-def get_container_config(remote, name):
-    rcontainer = remote + ":" + name if remote else name
+def get_container_config(container):
     cfg, _, _ = lxc(
-        ['config', 'show', rcontainer], ofmt=FMT_YAML,
-        fmsg="Failed to get config for '%s'. Does it exist?" % rcontainer)
+        ['config', 'show', container], ofmt=FMT_YAML,
```

Comment:
What's converting container object to a string?

Reply:
see above.